src/test/resources/org/nest/nestml/_cocos/equationsOnlyForStateVariables/valid.nestml

```diff
@@ -1,17 +1,17 @@
 neuron iaf_neuron_ode:
 
   state:
-    V mV
+    Volt mV
     G ms
   end
 
   equations:
     G' = e * t
-    V' = -1/Tau * V + 1/C_m
+    Volt' = -1/Tau * Volt + 1/C_m
   end
```

Comment:
Rather make this V_m (`Volt` is the name of the unit. `voltage` would work here, but `V_m` is much more common)